secrets/org.osbuild.rhsm

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+import json
+import os
+import sys
+import glob
+
+def main():
+    rhsm_secrets = {
+        'ssl_ca_cert': "/etc/rhsm/ca/redhat-uep.pem",
+        'ssl_client_key': "",
+        'ssl_client_cert': ""
+    }
+
+    keys = glob.glob("/etc/pki/entitlement/*-key.pem")
+    for key in keys:
+        cert = key.rstrip("-key.pem") + ".pem"
+        if os.path.exists(cert):
+            rhsm_secrets['ssl_client_key'] = key
+            rhsm_secrets['ssl_client_cert'] = cert
+            json.dump(rhsm_secrets, sys.stdout)
+            return 0
```

Comment:
Is there ever a reason to have more than one key in that directory? Is the name `entitlement` somehow reserved by Red Hat for their keys?

Anyway, this is an implementation issue, the API is sound, so fine with me. I just wondered how we find the key when there is more stuff in that directory than just one key-pair?